packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -86,6 +95,8 @@ export class GutenbergEditorPage {
 		// takes too long here, so the most reliable alternative is the title being
 		// visible.
 		await frame.waitForSelector( selectors.editorTitle );
+		// Once https://github.com/Automattic/wp-calypso/issues/57660 is resolved,
+		// the next line should be removed.
```

Comment:
Not necessarily related to this PR, though I think even after #57660 is fixed this should remain in case.